cloudinit/sources/DataSourceWSL.py

```diff
@@ -329,8 +329,12 @@ def _get_data(self) -> bool:
         # That's the reason for not using util.mergemanydict().
         merged: dict = {}
         overridden_keys: typing.List[str] = []
+        user_tags = None
         if user_data:
             merged = user_data
+            user_tags = (
+                merged.get("landscape", {}).get("client", {}).get("tags")
```

Comment:
Our agent data overriding user-data policy in WSL leads us down this path where we may start defining multiple exceptions to the "rule". If we end up coming up with a second case were other user-data keys are preferred over agent data keys, we may want to define a separate  userdata_override_paths dict or list or a separate function that extracts a dict of userdata_overrides from the original data so we have one place that defines these expected userdata paths.

Reply:
Given that every computer needs a different landscape.client.computer_title would we expect to override this with user-data too?

Reply:
Great thanks!